src/tests/SeqTests.fs

```diff
@@ -667,13 +667,25 @@ let ``Seq.truncate works``() =
     xs |> Seq.truncate 2
     |> Seq.last
     |> equal 2.
-    // Seq.truncate shouldn't throw an exception if there're not enough elements 
+    // Seq.truncate shouldn't throw an exception if there're not enough elements
     try xs |> Seq.truncate 20 |> Seq.length with _ -> -1
     |> equal 5
 
 [<Test>]
 let ``Seq.where works``() =
     let xs = [1.; 2.; 3.; 4.; 5.]
     xs |> Seq.where (fun i -> i <= 3.)
-    |> Seq.length 
-    |> equal 3
\ No newline at end of file
+    |> Seq.length
+    |> equal 3
+
+[<Test>]
+let ``Seq.except works``() =
+    Seq.except [2] [1; 3; 2] |> Seq.last |> equal 3
+    Seq.except [2] [2; 4; 6] |> Seq.head |> equal 4
+    List.except [1] [1; 1; 1; 1] |> List.isEmpty |> equal true
```

Comment:
Just for the sake of consistency, would you mind putting the `List.except` assertions in a different tests in the `ListTests.fs` file. It'd be great if you can also add a test for `ArrayTests.fs` (just copying the same test and replacing `List` with `Array` should be fine).